webpack/client.js

```diff
@@ -35,8 +38,11 @@ const clientConfig = {
   },
   plugins: [
     new MiniCssExtractPlugin({
-      filename: '../styles/[name].css',
-      chunkFilename: '../styles/[id].css',
+      filename: env.isDev ? 'styles/[name].css' : 'styles/[name].[contenthash].css',
+      chunkFilename: env.isDev ? 'styles/[id].css' : 'styles/[id].[contenthash].css',
+    }),
+    new WebpackManifestPlugin({
+      publicPath: '',
```

Comment:
Is setting an explicit `publicPath` value to empty string necessary?

Reply:
It seems this is still necessary [using the plugin with Webpack 5](https://stackoverflow.com/a/65272040).